### PR TITLE
🎨 Fix up the NuGet meta info

### PR DESCRIPTION
**Pull request by PureKrome**
image

### CODE REVIEW

File: artifacts/SimpleAuthentication.Core.1.2.3.nupkg
Comment (phillip-haydon) [inline artifacts/SimpleAuthentication.Core.1.2.3.nupkg:1]: Remove, maybe add *.nupkg to the got ignore.

File: artifacts/SimpleAuthentication.ExtraProviders.1.2.3.nupkg
Comment (phillip-haydon) [inline artifacts/SimpleAuthentication.ExtraProviders.1.2.3.nupkg:1]: Same same.

File: artifacts/SimpleAuthentication.Core.1.2.3.nupkg
Comment (PureKrome) [reply artifacts/SimpleAuthentication.Core.1.2.3.nupkg:1]: 🔧 Fixed. nuked.

File: artifacts/SimpleAuthentication.ExtraProviders.1.2.3.nupkg
Comment (PureKrome) [reply artifacts/SimpleAuthentication.ExtraProviders.1.2.3.nupkg:1]: 🔧 Fixed. nuked.

**Pull request merged**